src/vscode-bicep/src/utils/OutputChannelManager.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import {
+  createAzExtOutputChannel,
+  IAzExtOutputChannel,
+} from "@microsoft/vscode-azext-utils";
+
+export class OutputChannelManager {
```

Comment:
Sorry I missed this last time - `_azExtOutputChannel` is also disposable. We should let `OutputChannelManager` inherit `Disposable` and register the output channel. The `OutputChannelManager` instance created in `extension.ts` must be registered as well.

Reply:
Ah thank you!

I updated it here:
https://github.com/Azure/bicep/commit/290c718d712e67316e28b712c948f105ddd84994

Took another look at .ts files. Hope I am not missing anything.